ace-link-dashboard.el

```diff
@@ -37,6 +37,7 @@
 (require 'avy)
 
 (declare-function dashboard-remove-item-under "dashboard" nil)
+(declare-function dashboard--current-section "dashboard" nil)
```

Comment:
I don't want to use [`dashboard--current-section`](https://github.com/emacs-dashboard/emacs-dashboard/blob/402011afef7560eb8d87d1ffd135f8b43aeec898/dashboard.el#L131) here because only will work with the "official" sections (recents, bookmarks, agenda... etc.). Besides, [`dashboard-remove-item-under`](https://github.com/emacs-dashboard/emacs-dashboard/blob/402011afef7560eb8d87d1ffd135f8b43aeec898/dashboard.el#L295) uses again that function.